config/api.yml

```diff
@@ -1517,6 +1545,21 @@
       :post:
       - :name: edit
       - :name: delete
+  :alert_status_states:
+    :description: alert status state
+    :options:
+    - :subcollection
+    :verbs: *gpd
+    :klass: MiqAlertStatusState
+    :subcollection_actions:
+      :post:
+      - :name: add
```

Comment:
while we can add alert_state_states to an /api/alert_states, it there a need to add an /api/alert_states to a provider or is that done by the environment ?